### PR TITLE
add more tempating posibility for Service managment.

### PR DESCRIPTION
**Pull request by tomaszmichalowski**
Solves Issue #47

### CODE REVIEW

File: charts/flipt/templates/service.yaml [hunk 1]
```diff
@@ -1,10 +1,33 @@
+{{- if .Values.service.enabled }}
```
Comment (GeorgeMac) [inline charts/flipt/templates/service.yaml:1]: As this (`service.enabled`) is a new property, could we perhaps make it so that it renders the service if this value is not defined. For  backwards compatibility.

As in:

```suggestion
{{- if (or (empty .Values.service.enabled) .Values.service.enabled))}}
```

**Review by GeorgeMac — CHANGES_REQUESTED** [review]
Thanks again for the contribution here! Just one small request for backwards compatibility.

File: charts/flipt/templates/service.yaml [hunk 2]
```diff
@@ -1,10 +1,36 @@
+{{- if (or (empty .Values.service.enabled) .Values.service.enabled))}}
```
Comment (GeorgeMac) [inline charts/flipt/templates/service.yaml:1]:
```suggestion
{{- if (or (empty .Values.service.enabled) .Values.service.enabled)}}
```
Was a mistake in the github suggestion I made. Apologies.

**Review by GeorgeMac — CHANGES_REQUESTED** [review]
I made a typo which is breaking it. My bad.

File: charts/flipt/templates/service.yaml (hunk 2 above)
Comment (GeorgeMac) [reply charts/flipt/templates/service.yaml:1]: @tomaszmichalowski just a heads up. One this fix is applied I think it is ready to merge (given tests pass).
I left it here for you to do the honours. Just to sanity check me.

**Comment by tomaszmichalowski** [comment]
Still need to work on this if statement

```The output of this statement depends on the value of .Values.service.enabled.

If .Values.service.enabled is true, then the or function will short-circuit and return true, and the if statement will evaluate to true.

If .Values.service.enabled is false, then the or function will evaluate (empty .Values.service.enabled) which will return true because .Values.service.enabled is empty (i.e., it does not exist or its value is an empty string). Therefore, the or function will return true, and the if statement will evaluate to true.

If .Values.service.enabled is not set or is set to a non-boolean value (e.g., a string), then (empty .Values.service.enabled) will also return true because it checks for the existence of the value. Therefore, the or function will return true, and the if statement will evaluate to true.

So, in summary, the if statement will evaluate to true if .Values.service.enabled does not exist, is an empty string, or is set to true.```

**Comment by tomaszmichalowski** [comment]
now works:
service.enabled = true -> service generated
service.enabled = false -> service disabled
service.enabled does not exist -> service generated

**Comment by GeorgeMac** [comment]
Thanks @tomaszmichalowski !
